pkgs/tools/misc/SP800-90B_EntropyAssessment/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bzip2
+, libdivsufsort
+, jsoncpp
+, openssl
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "SP800-90B_EntropyAssessment";
+  version = "1.1.6";
+
+  src = fetchFromGitHub {
+    owner = "usnistgov";
+    repo = "SP800-90B_EntropyAssessment";
+    rev = "v${version}";
+    hash = "sha256-KZQ7kC0PbBkjLEQZIqYakQ91OvCxruhdfUwiRHtno3w=";
+  };
+
+  buildInputs = [ bzip2 libdivsufsort jsoncpp openssl mpfr ];
+
+  sourceRoot = "source/cpp";
+
+  makeFlags = lib.optionals stdenv.hostPlatform.isAarch64 [
```

Comment:
Are these needed anymore?

Reply:
When i comment out the makeFlags then the cross-compilation with `nix-build -A pkgsCross.aarch64-multiplatform.SP800-90B_EntropyAssessment` does fail. So for usage on aarch64-linux we would still those.

Reply:
Do this instead

```
  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
  "CROSS_COMPILE=${stdenv.cc.targetPrefix}"
  "ARCH=${stdenv.hostPlatform.linuxArch}"
  ]);

Reply:
I have changed it as you suggested. This also builds successfully.